### PR TITLE
Updates on parameter names and checking if key exists.

### PR DESCRIPTION
**Pull request by nicoletadscux**

### CODE REVIEW

File: exercise_1_solution.py [hunk 1]
```diff
@@ -84,7 +83,5 @@ def get_non_matching_entries(data: list, filter_key: str, filter_value: str):
     :returns filter_value: The matching filter
     :type filter_value: string
     """
-    all_keys: set = {k for d in data for k in d.keys()}
-    if filter_key in all_keys:
-        logger.info('Return all non-matching data for filter key and filter value: ', filter_key, filter_value)
-        return [d for d in data if d[filter_key] is not filter_value]
+    logger.info('Return all non-matching data for filter key and filter value.')
+    return [d for d in data if filter_key in d and d[filter_key] == filter_value]
```
Comment (khmel-aleh) [inline exercise_1_solution.py:87]: `==` Is it correct?)

File: exercise_1_solution.py [hunk 2]
```diff
@@ -67,13 +68,11 @@ def get_matching_entries(data: list, filter_key: str, filter_value: str):
     :returns filter_value: The matching filter
     :type filter_value: string
     """
-    all_keys: set = {k for d in data for k in d.keys()}
-    if filter_key in all_keys:
-        logger.info('Return all matching data for filter key and filter value: ', filter_key, filter_value)
-        return [d for d in data if d[filter_key] is filter_value]
+    logger.info('Return all matching data for filter key and filter value.')
```
Comment (khmel-aleh) [inline exercise_1_solution.py:71]: I'd add info about filter:
```
logger.info("Return all matching data for filter key '%s' and filter value '%s", filter_key, filter_value)
```

**Comment by khmel-aleh** [comment]
And I propose to add a few calls for testing purposes: with correct and negative scenarios

File: exercise_1_solution.py (hunk 1 above)
Comment (khmel-aleh) [reply exercise_1_solution.py:87]: Ok, if I run `test_filter_not_matching_accept()` I expect that there will not be items with `Result = "Accept"` But I see)
```
INFO:root:Return all non-matching data for filter key: 'Result' and filter value: 'Accept'.
INFO:root:Entries which do not match filter for Result Accept are: [{'Key': 'A001', 'Result': 'Accept', 'Service': 'Equifax'}, {'Key': 'A003', 'Result': 'Accept', 'Service': 'Equifax'}]
```

**Comment by khmel-aleh** [comment]
And please check linter
image
